SatHunter/SettingsView.swift

```diff
@@ -1,114 +1,129 @@
 import SwiftUI
 
 func getBtId(requestNew: Bool = false) -> UUID {
-  if let stored = UserDefaults.standard.string(forKey: "BTID") {
-    if let uuid = UUID(uuidString: stored) {
-      if !requestNew {
-        return uuid
-      }
+    if let stored = UserDefaults.standard.string(forKey: "BTID") {
```

Comment:
some files are formatted with 4-space indentation while some others are with 2-space... 

I remember using SwiftFormat and did not bother tweaking any its settings. Which formatter do you use and which preset ?